NewEvaluation.css

```diff
@@ -180,6 +148,52 @@ h3{
 
 }
 
+/*---------------------- TextArea Box---------------------------*/
+
+.user-textarea textarea{
+	width:100%;
+	height: 100px;
+}
+
+
+
+/*------------------- Legend Box ------------------*/
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
```

Comment:
Foound out why I couldn't do the 33% then 50% 50%, the drop down box menu has a initial padding of 3px around. Wanted also to know if there is a way to make the width with % values or VW but also add and subtract pixels ? something like 50% -3px in this case